config/config.yml

```diff
@@ -323,48 +323,73 @@ BLACKLISTED_MULTIMEDIA_SRCS: []
-  "drop-initial-before-align", "drop-initial-size", "drop-initial-value", "elevation", "empty-cells", "filter", "fit", "fit-position", "float",
-  "float-offset", "font", "font-effect", "font-emphasize", "font-emphasize-position", "font-emphasize-style", "font-family", "font-size", "font-size-adjust",
-  "font-smooth", "font-stretch", "font-style", "font-variant", "font-weight", "grid-columns", "grid-rows", "hanging-punctuation", "height", "hyphenate-after",
-  "hyphenate-before", "hyphenate-character", "hyphenate-lines", "hyphenate-resource", "hyphens", "icon", "image-orientation", "image-resolution", "ime-mode",
-  "include-source", "inline-box-align", "layout-flow", "left", "letter-spacing", "line-break", "line-height", "line-stacking", "line-stacking-ruby",
-  "line-stacking-shift", "line-stacking-strategy", "mark", "mark-after", "mark-before", "marks", "marquee-direction", "marquee-play-count",
-  "marquee-speed", "marquee-style", "max-height", "max-width", "min-height", "min-width", "move-to", "nav-down", "nav-index", "nav-left", "nav-right",
-  "nav-up", "opacity", "orphans", "page",
-  "page-policy", "phonemes", "pitch", "pitch-range", "play-during", "position", "presentation-level", "punctuation-trim",
-  "quotes", "rendering-intent", "resize", "rest", "rest-after", "rest-before", "richness", "right", "rotation", "rotation-point", "ruby-align", "ruby-overhang",
-  "ruby-position", "ruby-span", "size", "speak", "speak-header", "speak-numeral", "speak-punctuation", "speech-rate", "stress", "string-set", "tab-side",
-  "table-layout", "target", "target-name", "target-new", "target-position", "top", "unicode-bibi", "unicode-bidi", "user-select",
-  "vertical-align", "visibility", "voice-balance", "voice-duration", "voice-family", "voice-pitch", "voice-pitch-range", "voice-rate", "voice-stress",
-  "voice-volume", "volume", "white-space", "white-space-collapse", "widows", "width", "word-break", "word-spacing", "word-wrap", "writing-mode", "z-index"]
-
-# we allow through any string that is just a-z plus dashes (no spaces), so the only keywords you need to specify would
-# be ones that have something else or url, if you want to allow url values
+# We use this list for properties which can take shorthand values and also have
+# subproperties or variants (ie, because "border" is in here, we allow through: 
+# border-right, border-bottom-left-radius, -moz-border-foo, etc)
+# It is slightly less secure so do NOT put any property in here if you aren't
+# sure all variations on it are okay!
+SUPPORTED_CSS_SHORTHAND_PROPERTIES: ["background", "border", "column", "cue",
+  "flex", "font", "layer-background", "layout-grid", "list-style", "margin",
+  "marker", "outline", "overflow", "padding", "page-break", "pause",
+  "scrollbar", "text", "transform", "transition"]
+
+SUPPORTED_CSS_PROPERTIES: ["-replace", "-use-link-source", "accelerator",
```

Comment:
Out of curiosity, any reason not to use the YAML block-style list?
```yaml
SUPPORTED_CSS_PROPERTIES:
- "-replace"
- "-use-link-source"
- "accelerator"
- etc
```
It might make reviewing future diffs a bit easier, since there's one element per line. I guess it could result in this file getting pretty large though... :thinking:

Reply:
No reason that I'm aware of! I like the idea of easier-to-read diffs, so I can change it.

Reply:
Actually, let me check I won't be making more work for Systems first...

Reply:
No I think it would be fine :)